sklearn/utils/estimator_checks.py

```diff
@@ -397,6 +398,39 @@ def check_dtype_object(name, Estimator):
 
 
 @ignore_warnings
+def check_dict(name, Estimator):
+    rnd = np.random.RandomState(0)
+    if name in ['SpectralCoclustering']:
+        return 0
```

Comment:
just "return", no 0.

The issues with `SpectralCoclustering` and `SpectralBiclustering` may be resolved in #6141